norbit-webapp/utils/mochAmplify.ts

```diff
@@ -0,0 +1,2 @@
+// utils/amplifyUtils.ts
+export const isMockAmplify = () => process.env["NEXT_PUBLIC_MOCK_AMPLIFY"] == "yes";
```

Comment:
What purpose does this serve now that the function has been inlined in a redux variable?

Reply:
no purpose